authentik/docker-compose.yml

```
@@ -44,6 +44,7 @@ services:
       - ./media:/media
       - ./custom-templates:/templates
       - ./data:/data
+      - ./media/custom.css:/web/dist/custom.css
```

Would be better to have this:
```suggestion
      - ./web/dist/custom.css:/web/dist/custom.css
```

---

`./web/dist` does not exist in the repo, it wil be created in a build process.
So what about placing the custom CSS file in `./custom.css` and binddit in the YAML file this way `./custom.css:/web/dist/custom.css`?